### DISCUSSION TITLE
ANSI style on powershell 7 when PSStyle is defined

### DISCUSSION THREAD

**Discussion by cderv** [jdhitsolutions/PSScriptTools#146] [Q&A]
Hi, 

I wanted the `-ansi` to work with `Show-Tree` but I understand the `$PSAnsiMap` is not loaded because I have some `$PSStyle` defined - not sure if this is something I loaded or default in Windows Terminal. 

My understanding is that `$PSAnsiMap` is not loaded to no conflict, but it seems `-ansi` doesn't use `PSStyle`. 

Is there any way to have the coloring works for functions that supports it ? 

thank you

**Comment by jdhitsolutions** [comment]
I will have to look at this again to see what I can do to make it compatible with PSStyle. I'm going to move this to an Issue.
